mundane-dsv/src/main/scala/com/ambiata/mundane/dsv/DelimitedValuesParser.scala

```diff
@@ -0,0 +1,68 @@
+package com.ambiata.mundane
+package dsv
+
+import scala.collection.JavaConversions._
+
+import org.apache.commons.csv.{CSVParser, CSVFormat}
+
+import scalaz._, Scalaz._
+
+/**
+ * Create a parser for delimited values based on
+ * different implementations and a choice of
+ *
+ *  - delimiter
+ *  - character used for quoting fields
+ *  - character used to escaped control characters (delimiter, quoteCharacter)
+ *
+ */
+object DelimitedValuesParser {
+
+  def createParser(delimiter: Char, quoteCharacter: Option[Char], escapeCharacter: Option[Char], implementation: DelimitedValuesParserImplementation): DelimitedValuesParser =
```

Comment:
I don't think it is a safe assumption that these things will always be configurable. Probably doesn't matter at the moment, just an observation that I don't think one set of configuration options will make a lot of sense long term.